ansible_base/models/organization.py

```diff
@@ -0,0 +1,30 @@
+"""Organization models."""
+from django.conf import settings
+from django.db import models
+
+from .common import UniqueNamedCommonModel
+
+
+class AbstractOrganization(UniqueNamedCommonModel):
+    """An abstract base class for organizations."""
+
+    class Meta:
+        abstract = True
+
+    description = models.TextField(
+        null=False,
+        default="",
+        help_text="The organization description.",
```

Comment:
@john-westcott-iv do these help_text strings need to be translated?

Reply:
Yes, we have a ton of strings that are going to need translation. That is likely going to be a large task at somepoint.

Reply:
Closing this since net-new files are translated and we will have to go back through and translate remaining files.